examples/audio/audio.rs

```diff
@@ -1,16 +1,30 @@
 use bevy::prelude::*;
 
 /// This example illustrates how to load and play an audio file
+
+struct Music(Handle<AudioSource>, bool);
+
 fn main() {
     App::build()
         .add_default_plugins()
         .add_startup_system(setup.system())
+        .add_system(play_music_system.thread_local_system())
         .run();
 }
 
-fn setup(asset_server: Res<AssetServer>, audio_output: Res<AudioOutput>) {
+fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
     let music = asset_server
         .load("assets/sounds/Windless Slopes.mp3")
         .unwrap();
-    audio_output.play(music);
+    commands.insert_resource(Music(music, false));
+}
+
+fn play_music_system(_world: &mut World, resources: &mut Resources) {
+    let audio_output = resources.get_thread_local::<AudioOutput>().unwrap();
+    let mut music = resources.get_mut::<Music>().unwrap();
+    if !music.1 {
+        audio_output.play(music.0);
+        println!("{:?}", audio_output);
+        music.1 = true;
+    }
```

Comment:
You can use a thread-local startup system, I tested that the following does work

```
use bevy::prelude::*;

/// This example illustrates how to load and play an audio file

fn main() {
    App::build()
        .add_default_plugins()
        .add_startup_system(setup.thread_local_system())
        .run();
}

fn setup(_world: &mut World, resources: &mut Resources) {
    let asset_server = resources.get_mut::<AssetServer>().unwrap();
    let audio_output = resources.get_thread_local::<AudioOutput>().unwrap();
    let music = asset_server
        .load("assets/sounds/Windless Slopes.mp3")
        .unwrap();
    audio_output.play(music);
}
```

Reply:
Good to know, (as of my latest commit) we are no longer using thread local systems to play audio, so this issue is inherently fixed.